### DISCUSSION TITLE
Software Engineering Survey on GitHub Discussions.

### DISCUSSION THREAD

**Discussion by raux** [qutebrowser/qutebrowser#5670] [General] [answered]
Do you find GitHub discussions useful as a communication channel? Maybe not? 
Hello Developers! I am Raula Gaikovina Kula, a Software Engineering researcher
https://raux.github.io/

Together with a team of researchers, we are interested in your thoughts about
using the GitHub Discussions Beta. We noticed that your project is currently using.

We have developed a simple 3 question survey that could take up to 5 mins to complete.
https://forms.gle/peBxiio8u99QvKMk6

Thank you in advance for your time and apologize if this is not a proper use of this channel.

@raux
Raula, PhD on behalf of Hideaki, Sebastian, Nicole and Christoph

**Comment by The-Compiler** [comment, marked as answer]
Why not - done!

**Comment by raux** [comment]
Thank you so much for your time. I will keep you informed of our findings in the future.
